lib/pipelines/code-pipeline.ts

```diff
@@ -2,57 +2,166 @@ import * as cdk from '@aws-cdk/core';
 import * as pipelines from '@aws-cdk/pipelines';
 import * as codepipeline from '@aws-cdk/aws-codepipeline';
 import * as actions from '@aws-cdk/aws-codepipeline-actions';
+import { Construct, StackProps } from '@aws-cdk/core';
```

Comment:
we should get aligned on when to spread imports and when to use `import * as`. We are inconsistent in the project currently.

Reply:
I mentioned elsewhere, I was using common conventions from my prev experience: wildcard after 5 imports.

Reply:
Appears to be resolved, as the number of imports before wildcards is 3. We need to look into making it a lint rule. So far I only observe a way to code that rule, so deferring for now.